### PR TITLE
ISPN-8983 Invalid Ickle query causes server exception

### PR DESCRIPTION
**Pull request by gustavonalle**
https://issues.jboss.org/browse/ISPN-8983

### CODE REVIEW

File: object-filter/src/main/java/org/infinispan/objectfilter/impl/syntax/parser/QueryRendererDelegateImpl.java [hunk 1]
```diff
@@ -355,10 +355,11 @@ public void predicateFullTextTerm(String term, String fuzzyFlop) {
       if (property.isEmpty()) {
          throw log.getPredicatesOnEntityAliasNotAllowedException(propertyPath.asStringPath());
       }
-      String comparisonValue = (String) parameterValue(term);
+      checkAnalyzed(property,true);
```
Comment (anistor) [inline object-filter/src/main/java/org/infinispan/objectfilter/impl/syntax/parser/QueryRendererDelegateImpl.java]: There's ```checkAnalyzed(property,true);``` inside the if statement already. Why do we need it here too?

Comment (gustavonalle) [reply object-filter/src/main/java/org/infinispan/objectfilter/impl/syntax/parser/QueryRendererDelegateImpl.java]: Where?

Comment (gustavonalle) [reply object-filter/src/main/java/org/infinispan/objectfilter/impl/syntax/parser/QueryRendererDelegateImpl.java]: Ok, saw it

**Comment by gustavonalle** [comment]
@anistor Updated

**Comment by anistor** [comment]
Merged. Thanks @gustavonalle !

**Pull request merged**